lib/Requestable.js

```diff
@@ -277,7 +283,9 @@ function getNextPage(linksHeader = '') {
 
 function callbackErrorOrThrow(cb, path) {
    return function handler(response) {
```

Comment:
This can be made safer by defining it as such: `return function handler({status, statusText, config:{method, url}})`. Though you should double check that the `config:{method, url}` part doesn't give "cannot read property `method` of `undefined`"